projects.go

```diff
@@ -0,0 +1,216 @@
+package tfe
```

Comment:
I noticed that the standard for the resource file naming seems to be singular (`project.go`, `project_integration_test.go`) instead of plural.